nucleus/payara-modules/opentracing-adapter/src/main/java/fish/payara/opentracing/OpenTracingService.java

```diff
@@ -131,11 +146,17 @@ public synchronized Tracer getTracer(String applicationName) {
 
     /**
      * Pass-through method that checks if Request Tracing is enabled.
-     * 
+     *
      * @return True if the Request Tracing Service is enabled
      */
     public boolean isEnabled() {
-        return Globals.getDefaultBaseServiceLocator().getService(RequestTracingService.class).isRequestTracingEnabled();
+        RequestTracingService requestTracingService = getFromServiceHandle(Globals.getDefaultBaseServiceLocator(),
```

Comment:
Is there a reason to not have these injected and kept in a field?

Reply:
Same reason as given above.
Injection attempts to start the service, something not necessarily desired when running on a Java SE client.